packages/terra-clinical-action-header/package.json

```diff
@@ -22,26 +22,26 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "devDependencies": {
-    "react-intl": "^2.3.0"
+    "react-intl": "^2.4.0"
   },
   "peerDependencies": {
-    "react": "^15.4.2 || ^16.2.0",
-    "react-dom": "^15.4.2 || ^16.2.0",
-    "terra-base": "^2.0.0",
-    "terra-button": "^1.0.0",
-    "terra-button-group": "^1.0.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.0.0",
+    "terra-button": "^2.0.0",
+    "terra-button-group": "^2.0.0",
     "terra-clinical-header": "^1.7.0",
-    "terra-icon": "^1.0.0"
+    "terra-icon": "^2.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-base": "^2.0.0",
-    "terra-button": "^1.0.0",
-    "terra-button-group": "^1.0.0",
+    "terra-base": "^3.0.0",
+    "terra-button": "^2.0.0",
```

Comment:
We should bump these to their minor versions to ensrue peer dependencies work ok. https://github.com/cerner/terra-core/commit/71f8b9870f142fb23bf757611e689b6038062b1e

Reply:
Minor versions are bumped: https://github.com/cerner/terra-clinical/pull/272/commits/f0dd30ac2b6797a81fe54590e16f6555bd5b561f